lib/widgets/svg_icon.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SvgIcon extends StatelessWidget {
+  const SvgIcon(
```

Comment:
Can we add default parameters to this constructor, so it will be possible to control color and size not only with theme:

const SvgIcon(
      this.iconPath,
      this.iconColor = null,
      this.iconWidth = null,
      this.iconHeight = null
);

Reply:
Or let's keep it as is, but rename to something like BottomBarIcon and use it only there?